src/NServiceBus.Core/Serializers/SimpleJson/EnumAwareStrategy.cs

```diff
@@ -34,5 +36,8 @@ public override object DeserializeObject(object value, Type type)
 
             return base.DeserializeObject(value, type);
         }
+
+        static EnumAwareStrategy enumAwareStrategy;
+        public static EnumAwareStrategy Instance => enumAwareStrategy ?? (enumAwareStrategy = new EnumAwareStrategy());
```

Comment:
why not assigning a new instance to the static field and just return the reference from the property? This expression wouldn't be threadsafe theoretically?

Reply:
I used the same approach as the PocoSerializationStrategy that is already in the SimpleJson. Feel free to change it to anything you seem more appropriate

Reply:
so is that just a very weird lazy initialization implementation then?

Reply:
Yes the same as used for the PocoSerializationStrategy